### ISSUE TITLE
file_picker_callback for images not working on IE

### ISSUE THREAD

**Issue by vidhib** [tinymce/tinymce#3558]
**I'd like to report a *bug*** 
I have add description of current behavior  on  tinymce fiddle: http://fiddle.tinymce.com/8Mfaab
I am using TinyMCE version 4.3.13 and IE11. 

Here's the summary of behavior. I have added it to fiddle as well.
I am using callback feature to insert images from my local machine. When I try to insert images, a file browser is opened and I read the file in base64 format and pass it to the file_picker_callback.
It works perfectly fine on all browsers except IE11. When I open the file picker for the first time on IE and select an image,  the insert/edit image dialog is not populated with the image in source field. It is empty. However if I close it and reopen it, image source is populated with image in base64 format(expected behavior).

So basically I have to click on insert image twice to insert an image into IE from my local machine.

Please look into this issue and suggest a workaround if possible

**Comment by jayarjo** [comment]
You seem to trigger click event first and then attach click listener:

```javascript
$('#upload').trigger('click');
$('#upload').on('change', function() {
...      
```

Maybe try the opposite. Also your approach obviously will work for small images only, since for big images base64 string might easily overflow browser capabilities. Consider slightly different approach from our demo: https://www.tinymce.com/docs/demo/file-picker/.

Anyway this doesn't seem to be TinyMCE issue, so I will close it.
